Corona-Warn-App/src/main/res/values-de/dcc_ticketing_strings.xml

```diff
@@ -2,8 +2,17 @@
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
     <!-- Not in Dcc ticketing allow list dialog-->
 
+    <!-- Error Handling -->
+    <!-- XTXT: Service Provider (without provider name) error message-->
+    <string name="dcc_ticketing_error_service_provider_error_no_name">Es trat ein Fehler bei der Verbindung mit dem Anbieter. Bitte kontaktieren Sie den Anbieter wenn das Problem weiterhin besteht.</string>
```

Comment:
Please change "Es trat ein Fehler bei der Verbindung mit dem Anbieter." to "Es trat ein Fehler bei der Verbindung mit dem Anbieter auf." ("auf" is missing at the end).